docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -10,8 +10,42 @@ document are written by the developer most familiar with the change made.
 The contribution should be included as part of the Iris Pull Request that
 introduces the change.
 
-The ``latest.rst`` and the past release notes are kept in 
-``docs/iris/src/whatsnew/``.
+The ``latest.rst`` and the past release notes are kept in
+``docs/iris/src/whatsnew/``. If you are writing the first contribution after
+an Iris release: **create the new** ``latest.rst`` by copying the content from
+``latest.rst.template`` in the same directory.
+
+Since the `Contribution categories`_ include Internal changes, **all** Iris
+Pull Requests should be accompanied by a "What's New" contribution.
+
+
+Git Conflicts
+=============
+
+If changes to ``latest.rst`` are being suggested in several simultaneous
+Iris Pull Requests, Git will likely encounter merge conflicts. In this
+situation is thought likely (large PR, high repo activity etc.):
+
+* PR author: Do not include a "What's New" entry. Mention in the PR text that a
+  "What's New" entry is pending
+
+* PR reviewer: Review the PR as normal. Once the PR is acceptable, ask that
+  a **new pull request** be created specifically for the "What's New" entry,
+  which references the main pull request and titled (e.g. for PR#9999):
+
+    What's New for #9999
+
+* PR author: create the "What's New" pull request
+
+* PR reviewer: once the "What's New" PR is created, **merge the main PR**.
+  (If any links in the "What's New" PR referenced new features, these should
+  now be working).
```

Comment:
Good point @stephenworsley . This workflow is an effort to avoid needing to rebase, and there's every chance that a rebase to the "What's New" PR might be followed by actual "What's New" corrections and it would all get pushed up.

I don't see an easy solution, and in the long term I'm personally in favour of @bouweandela 's proposal (https://github.com/SciTools/iris/issues/3837#issuecomment-691956633), so in the meantime do you think it would be OK to just make specific reference to Travis' linkcheck test as being the thing that `should now be working`?

Reply:
That sounds fair to me.

Reply:
544d324c4cc638fbccc06cbcde9e05a170a1a205